recipes/default.rb

```diff
@@ -30,17 +32,17 @@
     end
   end
 
-  yum_version_lock node['osl-docker']['package_cli_name'] do
+  yum_version_lock node['osl-docker']['package']['package_name'] do
```

Comment:
Why was this changed?

Reply:
Cent OS 7 refused to version lock the cli package if they were in the other order for some odd reason.

Reply:
Strange, but I confirm it as well.